lib/components/widget-contact-info-component.js

```diff
@@ -2,9 +2,9 @@
 
 import { By } from 'selenium-webdriver';
 
-import BaseContainer from '../base-container.js';
+import AsyncBaseContainer from '../async-base-container.js';
```

Comment:
All the methods for a container using AsyncBaseContainer should be `async` and use `await`